python/activity_stream/reply_list.py

```diff
@@ -418,6 +420,7 @@ def _on_reply_clicked(self, note_id):
             self.__small_overlay.show()
             if reply_dialog.exec_() == QtGui.QDialog.Accepted:
                 self._data_manager.rescan()
+                log_metric_created_reply(self._bundle, "Reply List")
```

Comment:
Should this be done in the `ReplyDialog`?

Reply:
Would that be another one then? There is a field for source. One would be "Reply List" the one your are mentionning would be "Reply Dialog" ?

Reply:
How do I know I'm in "Reply List" from within the QT Dialog class ?